SPECS/nlohmann-json/nlohmann-json.spec

```diff
@@ -0,0 +1,55 @@
+Summary:        Modern C++11 JSON library
+Name:           nlohmann-json
```

Comment:
Please also update `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md ` for both packages.

Reply:
Fixed alongside other comments from this pass, thanks!